AllReadyApp/Web-App/AllReady/Controllers/AccountController.cs

```diff
@@ -411,5 +411,29 @@ private static void RetrieveFirstAndLastNameFromExternalPrincipal(ExternalLoginI
             firstName = array[0];
             lastName = array[1];
         }
+        
+        //Area Names
+        private const string ADMIN_AREA = "Admin";
+
+        //Controller Names
+        private const string HOME_CONTROLLER = "Home";

```

Comment:
@MisterJames, can we change title of #340 to use `nameof()` instead of magic strings? We should not be using constants anywhere we can avoid them (and I like to avoid them at all costs).